src/pages/home/HeaderView.tsx

```diff
@@ -202,7 +202,7 @@ function HeaderView({report, personalDetails, parentReport, parentReportAction,
     const brickRoadIndicator = ReportUtils.hasReportNameError(report) ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : '';
     const shouldShowBorderBottom = !isTaskReport || !isSmallScreenWidth;
     const shouldDisableDetailPage = ReportUtils.shouldDisableDetailPage(report);
-
+    const shouldUseGroupTitle = report?.reportName !== '';
```

Comment:
```suggestion
    const shouldUseGroupTitle = isGroupChat && !!report?.reportName;
```
- Limit this to group chat
- If the reportName is `undefined` we should not use full title (checking for truthy is safer than checking for empty string)